lms/templates/instructor/instructor_dashboard_2/instructor_dashboard_2.html

```diff
@@ -105,6 +98,9 @@
         <% dname = section_data['section_display_name'] %>
         <li class="nav-item"><a href="" data-section="${ section_data['section_key'] }">${_(dname)}</a></li>
       % endfor
+      %if analytics_dashboard_url:
```

Comment:
If any other "sections" are added they will come before this link and jumble the existing order. Is that intentional?